tests/phpunit/payment-methods/class-wc-stripe-upe-payment-method-pay-by-bank-test.php

```
@@ -0,0 +1,55 @@
+		$this->assertNotContains( WC_Stripe_Currency_Code::UNITED_STATES_DOLLAR, $currencies );
+	}
+
+	/**
+	 * Test that is_available_for_account_country() returns true when SUPPORTED_ACCOUNT_COUNTRIES
+	 * is empty (Pay by Bank has broad merchant country support; Stripe enforces at the account level).
+	 *
+	 * @dataProvider provide_test_is_available_for_account_country
+	 */
+	public function test_is_available_for_account_country( string $account_country, bool $expected_result ): void {
+		$this->run_is_available_for_account_country_test( WC_Stripe_UPE_Payment_Method_Pay_By_Bank::class, $account_country, $expected_result );
+	}
```

PHPMD (2.15.0)

[error] The method test_is_available_for_account_country is not named in camelCase. (undefined)

(CamelCaseMethodName)